go/pkg/trainingjober/trainingjober.go

```diff
@@ -0,0 +1,62 @@
+// trainingjober is a package for managing a TrainingJob
+package trainingjober
+
+import (
+	"github.com/PaddlePaddle/cloud/go/pkg/apis/paddlepaddle/v1alpha1"
+	trainingJobClient "github.com/PaddlePaddle/cloud/go/pkg/client/clientset/versioned"
+	"k8s.io/client-go/kubernetes"
+	"time"
+)
+
+const (
+	retryTimes            = 5
+	convertedTimerTicker  = 10 * time.Second
+	confirmResourceTicker = 5 * time.Second
+)
+
+type trainingJobEventType string
+
+const (
+	trainingJobEventDelete trainingJobEventType = "Delete"
+	trainingJobEventModify trainingJobEventType = "Modify"
+)
+
+type trainingJobEvent struct {
+	// pet is the TrainingJobEventType of TrainingJob
+	pet trainingJobEventType
+	// The job transfer the information fo job
+	job *v1alpha1.TrainingJob
+}
+
+// TrainingJober is to manager a specific TrainingJob
```

Comment:
I think we need a `TrainingjobController` to manager a TrainingJob.

Reply:
We have a controller which @m3ngyang committed in another PR https://github.com/PaddlePaddle/cloud/pull/565. We use controller to manager all TrainingJob of a specific cluster and use TrainingJober to manager a specific TrainingJob.